content/blog/survey-02/index.md

```diff
@@ -0,0 +1,221 @@
+We will try to get this done for the next newsletter.
+
+## What is going well
+
+![What is going well wordcloud](like.png)
+
+This was a free-text question. The above is a word cloud of the answers with some obvious words like "game" or "newsletter" removed.
+Note that the inclusion of the word "AI" is misleading, as it was only mentioned in answers that read similar to 
+"I like that we don't use AI, please don't use LLMs".
+
+Going through the feedback by hand, common things readers enjoy about the newsletter are:
+- A good mix of content
+- Very open to contributions
+- Small-scale games are featured, not just success stories or technical articles
+
+## What needs to be improved
+
+![What needs to be improved wordcloud](improve.png)
+
+Another free-text question. The feedback here is fairly diverse. The most common complaints we already mentioned in previous sections are:
+- Add an email subscription
+- Improve the ease of contributing
+
+Additionally, many people feel like the "Games" section reads more like an advertisement than an article aimed at other game developers.
+
+Among the more unique suggestions were:
+- Conduct interviews
+- Have a stronger sense of personality in the writing
+- Make the newsletter more consistent in timing and quantity
+- Have more editors to not overburden the current ones
+- Have some more clarity of purpose
+
+
+## Comments
+
+![Comments wordcloud](comment.png)
+
+This last free-text question was meant for any additional comments readers might have. 
+The word cloud above is dominated by one sentence: "Thank you for your work". Thank you very very much for your kind words! 
+We are working on this newsletter in our free time because we love the community and Rust game development, so reading this means a lot to us.
+
+## Correlations
+
+We were interested in how the responses to some questions correlated with how much people had already contributed to the newsletter.
+Long story short: it seems like there is no significant correlation between how much people contributed and how they answered the other questions.
+
+Let's look at the correlations in turn now.
+Note that all the following plots are jittered to make the data more readable.
+
+
+### Excitement By Contributions
+
+![Excitement correlation](excitement_contributions.png)
+
+We hypothesized that people who contributed more to the newsletter would be more excited about it. 
+We found no evidence for this (the p-value of a Jonckheere-Terpstra test for increasing trend is 0.986). 
+Based on the plot above, we then hypothesized that the opposite might be true, namely that frequent contributors are less excited about the newsletter.
+This actually might be the case (p-value is 0.021), but do not take this as a strong result.
+It is a posthoc hypothesis and the resulting p-value is not very low considering the number of tests we run in this analysis. 
```

Comment:
Either my understanding of p-values is mistaken, or it should be `the resulting p-value is very low`?

Reply:
The phrasing here is correct. 0.021 is not that low considering we ran three tests.